### DISCUSSION TITLE
Reset Global Settings for debugging

### DISCUSSION THREAD

**Discussion by noodles4694** [elgatosf/elgato#99] [Help] [answered]
Hi,

I have a plugin written with the Javascript SDK. It works fine on my computer (dev machione) but if someone downloads it from the marketplace they have an issue with one of the features. The second time they start their computer it works fine, and from that point on there will never be an issue on that PC.

I believe that there is an issue with initialising the plugin from a fresh install (where GlobalSettings do not exist) but I can't seem to ever replicate that "New" state on my dev machine, is there a way to remove ALL data for a plugin including global settings so that when i add the plugin it is the same as a new install on the computer ?

Any help would be appreciated.

**Comment by noodles4694** [comment, marked as answer]
Just for other developers: I managed to find a way to do it, if you navigate in the registry to HKEY_CURRENT_USER/SOFTWARE/Elgato Systems GmbH/StreamDeck  you should see a REG_SZ entry with the name of your plugin UUID twice, if you delete that key it deletes the local starage for your plugin, it will be recreated on restart

**Comment by tiptronic** [comment]
Hey @noodles4694 - what is wrong with un-installing/re-installing the plugin? That's the best way to find out what's going wrong on a fresh install
